sbi/src/sbi.rs

```diff
@@ -283,3 +283,8 @@ pub unsafe fn ecall_send(msg: &SbiMessage) -> Result<u64> {
 
     msg.result(a0, a1)
 }
+
+#[cfg(not(any(target_arch = "riscv64", target_os = "none")))]
+unsafe fn ecall_send(_msg: &SbiMessage) -> Result<u64> {
+    panic!("ecall_send called");
+}
```

Comment:
This (and the the changes to `api/pmu.rs`) belong in the previous commit.

Reply:
Moved.